go/service/chat_local.go

```diff
@@ -63,6 +63,22 @@ func (h *chatLocalHandler) NewConversationLocal(ctx context.Context, trip chat1.
 	return id, err
 }
 
+func (h *chatLocalHandler) CompleteAndCanonicalizeTlfName(ctx context.Context, tlfName string) (res keybase1.CanonicalTlfName, err error) {
+	// Append username in case it's not present. We don't need to check if it
+	// exists already since CryptKeys calls below transforms the TLF name into a
+	// canonical one.
+	tlfName = tlfName + "," + string(h.G().Env.GetUsername())
```

Comment:
@mmaxim if the username is already in there, it gets appended too. It relies on KBFS to take care of canonicalize the TLF name (including deduplication). This also covers the case where you have `t_alice, __songgao__@twitter`, and checking if one of usernames is `songgao` won't actually work because `__songgao__@twitter` is essentially the same user as `songgao`.